onnxruntime/core/providers/nnapi/nnapi_builtin/builders/model_builder.h

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+#include <onnx/onnx_pb.h>
+#include <unordered_set>
+
+#include "core/providers/nnapi/nnapi_builtin/model.h"
+#include "core/providers/nnapi/nnapi_builtin/nnapi_lib/NeuralNetworksWrapper.h"
+#include "op_builder.h"
+#include "shaper.h"
+
+namespace onnxruntime {
+namespace nnapi {
+
+class ModelBuilder {
+ public:
+  using Index = uint32_t;
+  using IndexSeq = std::vector<Index>;
+  using Shape = Shaper::Shape;
```

Comment:
nit: Not a huge fan of 'using' statements like this. You're only saving a couple of characters and for a developer reading the code knowing something is a vector can be helpful.

Reply:
Removed Index  and IndexSeq, will keep shape for now